python/pyspark/mllib/clustering.py

```diff
@@ -90,6 +90,14 @@ class KMeansModel(Saveable, Loader):
     ...     rmtree(path)
     ... except OSError:
     ...     pass
+
+    >>> data = array([-383.1, 31.2, 32.2, 367.3]).reshape(2, 2)
+    >>> model = KMeans.train(
+    ...     sc.parallelize(data), 3, maxIterations=0, runs=1, initializationMode="random",
```

Comment:
By remove unnecessary parameters, I meant to remove: runs, initializationMode, initializationSteps, epsilon.